src/mixins/filing-template-mixin.ts

```diff
@@ -203,7 +218,7 @@ export default class FilingTemplateMixin extends Vue {
         name: FilingTypes.ALTERATION,
         certifiedBy: this.getCertifyState.certifiedBy,
         date: this.getCurrentDate, // "absolute day" (YYYY-MM-DD in Pacific time)
-        folioNumber: this.getFolioNumber
+        folioNumber: this.getFolioNumber || ''
```

Comment:
So it can't be null? Please add a comment (so someone doesn't refactor this later and break things).

Reply:
hmm, yea. I found this when filing. The back end returns 'NoneType on String' error or something if null. 
A better fix here, is to initialize it i think. Will fix this.

Reply:
Fixed 👍